perses/annihilation/relative.py

```diff
@@ -1,888 +1,1968 @@
-import simtk.openmm as mm
-from simtk import unit
+import sys
+import simtk.openmm as openmm
 import simtk.openmm.app as app
+import simtk.unit as unit
```

Comment:
When possible, we want to migrate `import` statements into just the methods that need them, so that imports are handled lazily. This not only speeds up imports, it also keeps code tidier, and makes it easier to compile the sphinx docs without requiring all the dependencies just to import the file.